js/intro.js

```diff
@@ -317,7 +330,14 @@ async function showSavedDetails() {
       }
     }
   } catch (err) {
-    alert('something went wrong');
-    location.href = 'https://realdevsquad.com';
+    window.addEventListener(
+      'click',
+      showToast(
+        '<i class="fa-solid fa-circle-exclamation"></i> something went wrong',
```

Comment:
Instead of passing icon here, you can pass type and let the function **showToast** handle which icon to show.